AmpegManualTestProject.txt

```diff
@@ -0,0 +1,104 @@
+--Ampeg Manual Test Project--
+https://ampeg.com/
+
+1. You decide what should be priority (to test) in the webapp.
+2. 6 manual test cases total.
+3. Ensure to include what 'should' be included in a manual test (title, summary, etc).
+4. Ensure that 5 tests consist of navigating to more than one page.
+5. Ensure that 2 tests consist of navigating to 3 or more pages.
+
+1:
+
+Title: Find Contact Info
+
+Summary: The user will navigate the website and discover the company's contact info.
+
+Pre-conditions: Start at the home page.
+
+Traceability: User is able to get contact info if needed.
+
+Steps: Be at the home page and click on "Support" at the top right. Navigate to the bottom and click on "Contact Us".
+
+Expected result: User should be on a page listed with the company's contact info.
+
+2:
+
+Title: Create account
+
+Summary: The user will make an account if they dont have one.
+
+Pre-conditions: The user doesn't have an account or wants to make another one and isn't logged in.
+
+Traceability: User is able to make an account.
+
+Steps: Start on the home page. Click on the icon at the top right that looks like a person. User should be able to enter

```

Comment:
This is a little ambiguous. You should say 'Click the Account icon'. 
Hover over the icon to get the tooltip - the name of the icon.